__mocks__/addAndRemove.js

```diff
@@ -9,7 +9,7 @@ const dom = new JSDOM(`
 const { document } = (new JSDOM('...')).window;
 
 const checkItem = localStorage.getItem('TaskToday');
```

Comment:
Please declare this variable only once in respect of the [DRY](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) principle.